src/commands/vote.ts

```diff
@@ -0,0 +1,18 @@
+import { FireMessage } from "../../lib/extensions/message";
+import { Language } from "../../lib/util/language";
+import { Command } from "../../lib/util/command";
+
+export default class extends Command {
+  constructor() {
+    super("vote", {
+      description: (language: Language) =>
+        language.get("VOTE_COMMAND_DESCRIPTION"),
```

Comment:
You never added this to en-US so it'll just return default

Reply:
i couldnt find where the languages were at, sorry

Reply:
> i couldnt find where the languages were at, sorry

look again then